code/game/objects/items/cigs_lighters.dm

```diff
@@ -297,6 +297,21 @@ CIGARETTE PACKETS ARE IN FANCY.DM
 		qdel(src)
 	. = ..()
 
+/obj/item/clothing/mask/cigarette/afterattack(mob/M, mob/user, proximity)
```

Comment:
There is already an afterattack() defined for cigarettes around line 180

Although double definitions like this would compile and work, it's considered bad practice and it makes code flow somewhat hard to follow

Reply:
made each def a proc and added a new def sorting it out.